jme3-core/src/main/java/com/jme3/scene/Mesh.java

```diff
@@ -1529,6 +1529,25 @@ public void addMorphTarget(MorphTarget target) {
     public MorphTarget[] getMorphTargets() {
         return morphTargets.getArray();
     }
+    
+        /**
+     * Get the name of all morphs in order.
+     * Morphs without names will be null
+     * @return an array
+     */
+    public String[] getMorphTargetNames() {
```

Comment:
Not sure why `getMorphTargetNames()` is needed? 
We can already access morph names using `getMorphTargets()`.